crates/sparrow-backend/src/logical_to_physical.rs

```diff
@@ -296,6 +375,24 @@ impl LogicalToPhysical {
     }
 }
 
+fn logical_to_physical_literal(
+    literal: &sparrow_logical::Literal,
+) -> error_stack::Result<ScalarValue, Error> {
+    match literal {
+        sparrow_logical::Literal::Null => todo!(),
+        sparrow_logical::Literal::Bool(_) => todo!(),
+        sparrow_logical::Literal::String(s) => Ok(ScalarValue::Utf8(Some(s.clone()))),
+        sparrow_logical::Literal::Int64(_) => todo!(),
+        sparrow_logical::Literal::UInt64(_) => todo!(),
+        sparrow_logical::Literal::Float64(_) => todo!(),
+        sparrow_logical::Literal::Timedelta {
+            seconds: _,
+            nanos: _,
+        } => todo!(),
+        sparrow_logical::Literal::Uuid(_) => todo!(),
```

Comment:
Leaving these to follow up in #787 ?

Reply:
Yeah. Either handle in #787 or as part of testing.